platform/security/idp/security-idp-client/src/main/java/org/codice/ddf/security/idp/client/IdpHandler.java

```diff
@@ -494,7 +494,12 @@ private String createAndSignAuthnRequest(boolean isPost, boolean wantSigned)
         issuer.setValue(spIssuerId);
         authnRequest.setIssuer(issuer);
 
-        authnRequest.setAssertionConsumerServiceURL(spAssertionConsumerServiceUrl);
+        if (idpMetadata.getAssertionConsumerServiceURLCheck()){
+            LOGGER.info("Setting optional AssertionConsumerServiceURL in AuthnRequest");
```

Comment:
The guidelines can be found here, for future reference: https://codice.atlassian.net/wiki/spaces/DDF/pages/7045131/Log+Levels